src/react-cancelable.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Observable } from "rx";
+
+const inTestingEnv = typeof jasmine !== "undefined";
```

Comment:
This is dead code.

The reason for it in the old code is that shallow rendering snapshots would always have the Disposable class in the snapshot instead of the decorated component. We should test that snapshots from enzyme shallow rendering works with this new implementation.

Reply:
Instead of doing this hack, the enzyme tests should be doing `shallow(<DecoratedComponent />).dive()`